README.md

```diff
@@ -86,26 +86,24 @@ jobs:
       with:
         creds: ${{ secrets.AZURE_CREDENTIALS }}
     - uses: azure/sql-action@v1
-      with:
-        server-name: REPLACE_THIS_WITH_YOUR_SQL_SERVER_NAME

```

Comment:
Since it's optional, let's not remove it
(It will be good to mention that it's optional)